08b/CompPhys_Lect08b_05_Exercises_SOLUTION.ipynb

```diff
@@ -503,7 +503,11 @@
     "# corrupted in the same way our time domain data was before\n",
     "plt.plot(\n",
     "    frequencies_CUT[data_length:-data_length],\n",
-    "    abs(data_frequency_domain[data_length:-data_length])\n",
+    "    data_frequency_domain[data_length:-data_length].real\n",
```

Comment:
Fix to the plot- legend said real/imaginary parts, but the plot was of absolute values